modules/apps/object/object-test/src/testFunctional/tests/PicklistExportImport.testcase

```diff
@@ -66,9 +66,61 @@ definition {
 	@description = "LPS-175645 - It's possible to add an object entry of a picklist with a translation of a previously picklist imported "
 	@priority = 3
 	test CanAddEntryWithTranslationPicklistImported {
+		task ("Given a picklist imported and an object created with a picklist field") {
+			Picklist.gotoPicklists();
 
-		// TODO pending implementation
+			Picklist.importPicklist(
+				picklistFileName = "simple_picklist",
+				picklistName = "ImportedPicklist175689");
+
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 175689",
+				objectName = "CustomObject175689",
+				pluralLabelName = "Custom Objects 175689");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 175689");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Field Picklist",
+				fieldPicklist = "ImportedPicklist175689",
+				fieldType = "Picklist");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject175689");
+		}
+
+		task ("When translating the picklist definition") {
+			Picklist.gotoPicklists();
 
+			Picklist.gotoPicklistView(picklistName = "ImportedPicklist175689");
+
+			Picklist.addLanguageOnPicklist(
+				itemName = "PicklistImportada175689",
+				newLanguage = "pt_BR");
+		}
+
```

Comment:
Add translation to Picklist Item too, this way it's possible to assert the item translated.